From: Robert Rostohar <Robert.Rostohar@arm.com>
Date: Mon, 21 Dec 2020 16:47:52 +0100
Subject: [PATCH 2/3] Remove MSPLIM setup in test main function

MSPLIM is typically setup in the startup file.

This removes the requirement on linker scripts with specific symbol names.
---
 app/main_ns.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/app/main_ns.c b/app/main_ns.c
index 9b63507..e3061c1 100644
--- a/app/main_ns.c
+++ b/app/main_ns.c
@@ -24,7 +24,6 @@
 #endif
 #include "tfm_log.h"
 #include "uart_stdout.h"
-#include "region.h"
 
 /**
  * \brief Modified table template for user defined SVC functions
@@ -141,13 +140,6 @@ __attribute__((noreturn))
 #endif
 int main(void)
 {
-#if defined(__ARM_ARCH_8_1M_MAIN__) || defined(__ARM_ARCH_8M_MAIN__)
-    /* Set Main Stack Pointer limit */
-    REGION_DECLARE(Image$$, ARM_LIB_STACK_MSP, $$ZI$$Base);
-    __set_MSPLIM((uint32_t)&REGION_NAME(Image$$, ARM_LIB_STACK_MSP,
-                                        $$ZI$$Base));
-#endif
-
     if (tfm_ns_platform_init() != ARM_DRIVER_OK) {
         /* Avoid undefined behavior if platform init failed */
         while(1);
-- 
